config/application.rb

```diff
@@ -1,5 +1,13 @@
 # frozen_string_literal: true
 
+require "dotenv"
+require "hanami/application"
+require_relative "../lib/soundeck/middleware/elapsed"
+
+# This is here until we establish a proper way to load settings through
+# Hanami::Application
+Dotenv.load
```

Comment:
I figured this was the simplest thing to do to keep this app working while we establish a nicer approach for Hanami proper.